sphinx_ansible_theme/static/css/ansible.css

```diff
@@ -484,3 +484,5 @@ tr .ansibleOptionLink {
 .rst-content div[class^="highlight"] pre {
     white-space: pre-wrap;
 }
+
+.rst-content dl dt { margin-bottom: 0; }
```

Comment:
I wonder if we only want this to affect direct `<dt>` children of `<dl>`. If yes, we could also apply this:
```suggestion
.rst-content dl>dt { margin-bottom: 0; }
```

Reply:
@webknjaz can you elaborate on your suggested edit? Where else would a <dt> be if not the children of a <dl> ?

Reply:
`dl dt` matches `<dl><something><else><dt>...</dt></else></something></dl>` (deeply nested)

`dl>dt` only matches `<dl><dt>...</dt></dl>` (direct)